extensions/interactions/NumericInput/directives/oppia-interactive-numeric-input.directive.ts

```diff
@@ -64,7 +69,11 @@ angular.module('oppia').directive('oppiaInteractiveNumericInput', [
             ctrl.submitAnswer(ctrl.answer);
           };
           ctrl.$onInit = function() {
-            ctrl.answer = '';
+            if (ctrl.populateWithSolution) {
```

Comment:
I wonder if we can pass in the solution instead of the directive trying to determine the solution.

Reply:
Yeah, that's what I'd tried initially, but that's too messy since every interaction's answer type is different. Also, the math interactions can't use any props to populate the answer so there would be a lot of unnecessary conditional passing of data if we pass it through props.

Reply:
Hm, can you please explain "math interactions can't use any props to populate the answer" part? Why can't math interactions use data passed via attributes?

Reply:
Alright, so we use the guppy editor for math interactions, which doesn't have any functionality to pass in data to initialize the editor with. There are separate API calls that need to be made in order to do that so currently, we have services that initialize the guppy editor with saved responses that makes these API calls to fill the math expressions in the editor when it loads up.
Does that make things clearer?

Reply:
Oh wait, so will making the same API calls here solve it (i.e. populate Guppy with what we want)?